pkg/sys/userns_idmap.go

```diff
@@ -0,0 +1,124 @@
+/*
+   Copyright The containerd Authors.
```

Comment:
Added comment to reference the original file

Reply:
I don't know if that is valid. I'd like other maintainers input on this

Reply:
Yes, this is a fairly common practice. We've had copies of Go internal code (tar functionality?), Docker has as well, I believe Kubernetes has copied files, too. You can't just change the license, but you can add your own copyright header since it is part of our codebase now. I wrote the original package in Moby anyway and I am a containerd author, so there's that too :)

Reply:
Thanks!